code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -142,74 +149,85 @@ Credit where due:
 /datum/game_mode/clockwork_cult/proc/greet_servant(mob/M) //Description of their role
 	if(!M)
 		return 0
-	var/greeting_text = "<br><b><span class='large_brass'>You are a servant of Ratvar, the Clockwork Justiciar.</span>\n\
-	Rusting eternally in the Celestial Derelict, Ratvar has formed a covenant of mortals, with you as one of its members. As one of the Justiciar's servants, you are to work to the best of your \
-	ability to assist in completion of His agenda. You may not know the specifics of how to do so, but luckily you have a vessel to help you learn.</b>"
+	var/greeting_text = "<br><span class='large_brass bold'>You are a servant of Ratvar!</span>\n\
+	<i>You have [round(time_to_prepare)] minutes to prepare before the crew is alerted of your presence and portals open across [station_name()].</i>\n\
+	<i>Need help? Look inside the box on your belt for an item called a clockwork slab, and click the Recollection button in the top left."
 	to_chat(M, greeting_text)
-	M.playsound_local('sound/ambience/antag/ClockCultAlr.ogg',100,0)
+	to_chat(M, sound('sound/ambience/antag/ClockCultAlr.ogg'))
```

Comment:
Last I checked, that was the proc to play a sound to everyone on the z. Is that something else, am I just confused?